obspy/signal/filter.py

```diff
@@ -38,7 +38,7 @@ def _filter(data, freqs, df, btype='band', ftype='butter',
 
 
 def bandpass(data, freqmin, freqmax, df, corners=4, zerophase=False,
-             axis=-1):
+             rp=None, rs=None, ftype='butter', axis=-1):
```

Comment:
Strictly speaking these are public methods so we kinda advertize them as unchanging, which means `axis` should not get shifted in its position. I know it's ugly to have it in the middle of the call syntax, but it might still be worth it over potential backwards compatibility breaches.

Reply:
The axis keyword was only added in #3291 which will land in the same release 1.5.0, so it should be OK.